akka-http/src/main/scala/akka/http/scaladsl/server/util/JwtKeyLoader.scala

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2022 Lightbend Inc.
+ */
+
+package akka.http.scaladsl.server.util
+
+import akka.http.scaladsl.server.util.JwtSupport.{JwtAsymmetricAlgorithmSecret, JwtNoneAlgorithmSecret}
+import pdi.jwt.{JwtAlgorithm, algorithms}
+
+import java.io.File
+import java.nio.file.Files
+import java.security.spec.{PKCS8EncodedKeySpec, X509EncodedKeySpec}
+import java.security.{KeyFactory, KeyPair}
+import javax.crypto.spec.SecretKeySpec
+import akka.pki.pem.{DERPrivateKeyLoader, PEMDecoder, PEMLoadingException}
+
+object JwtKeyLoader {
```

Comment:
Once we are done with the API we should go through and mark as much as possible as internal API, possibly just the directives should be user/public API